streams-activitypub-graph/src/test/scala/org/apache/streams/activitypub/graph/test/ActivityPubGraphTestSuiteExtension.scala

```diff
@@ -0,0 +1,82 @@
+package org.apache.streams.activitypub.graph.test
+
+import org.apache.http.client.utils.URIBuilder
+import org.apache.http.impl.NoConnectionReuseStrategy
+import org.apache.jena.fuseki.main.FusekiServer
+import org.apache.jena.query.Dataset
+import org.apache.jena.query.DatasetFactory
+import org.apache.jena.riot.RDFDataMgr
+import org.apache.juneau.rest.client.RestClient
+import org.apache.streams.activitypub.graph.config.WebfingerGraphImplConfig
+import org.apache.streams.activitypub.graph.impl.WebfingerGraphImpl
+import org.apache.streams.activitypub.graph.test.config.ActivityPubGraphTestSuiteExtensionConfig
+import org.apache.streams.config.ComponentConfigurator
+import org.junit.jupiter.api.extension.AfterAllCallback
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.ParameterContext
+import org.junit.jupiter.api.extension.ParameterResolver
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import java.net.URL
+
+object ActivityPubGraphTestSuiteExtension {
+  private final val configurator: ComponentConfigurator[ActivityPubGraphTestSuiteExtensionConfig] = new ComponentConfigurator(classOf[ActivityPubGraphTestSuiteExtensionConfig])
+  final val config: ActivityPubGraphTestSuiteExtensionConfig = configurator.detectConfiguration()
+}
+
```

Comment:
a junit5 extension bound to a test suite that runs an embedded fuseki loaded with test dataset(s) available to all test cases